content/attacks/posts/2016-03-14-ShapeShift.md

```diff
@@ -0,0 +1,49 @@
+---
+date: 2016-03-14
+target-entities: ShapeShift
+entity-types: Exchange
+attack-types:
+  - Insider Attack
+  - Infrastructure Attack
+  - Wallet Hack
+title: "ShapeShift Exchange Hacked for $230000"
+loss: 230000
+---
+
+## Summary
+
+From March 14 to April 9, 2016, there were 3 hacks of the Swiss ShiftShape crypto exchange.
+The first hack was made by an employee of the exchange. Then he left a backdoor in the system and sold access to a hacker, who committed two more hacks.
```

Comment:
In the first sentence, I guess the word `hack` is unappropriate, and please provide more details, for example `stole the keys of exchange's hot wallet` or something. The second sentence is also can be extended with technical details.